lib/version.rb

```diff
@@ -1,3 +1,3 @@
 module SendGridWebApi
-  VERSION = "0.0.8"
+  VERSION = "0.0.9"
```

Comment:
This is a backwards breaking change since we're now required to use tokens
```suggestion
  VERSION = "0.1.0"
```